backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/optimizations/CallGraphBuilder.kt

```diff
@@ -111,31 +109,6 @@ internal class CallGraphBuilder(val context: Context,
     private inline fun DataFlowIR.FunctionBody.forEachCallSite(block: (DataFlowIR.Node.Call) -> Unit) =
             nodes.forEach { node ->
                 when (node) {
-                    // TODO: OBJC-CONSTRUCTOR-CALL
-                    is DataFlowIR.Node.NewObject -> {
```

Comment:
Is `NewObject` still possible here? Otherwise consider assertion

Reply:
`NewObject` is a subtype of `Call`, before the PR it required special handling, now doesn't.